mdbenchmark/analyze.py

```diff
@@ -177,19 +98,26 @@ def analyze(directory, plot, ncores, output_name):
     df.to_csv(output_name)
 
     if plot:
+        console.warn(
```

Comment:
Are we using `console.warn` because we assume that users expect `mdbenchmark analyze --plot` to do something? I'd rather use `console.error` and exit, but can relate to the backwards compatibility.

Maybe we should announce a planned deprecation? I do not think keeping the feature for a longer time is that great.

Reply:
yes, currently this is still usable and I think it makes sense to keep it this way for now until the new version has been properly tested.

Reply:
why not turn this into a console info and keep it. As long as this internally uses `plot_over_group` there isn't much overhead.

Reply:
So we go with something like `console.info('DEPRACATED Please use xyz in the future')`!